src/OpenLoco/src/Windows/Construction/ConstructionTab.cpp

```diff
@@ -136,10 +136,10 @@ namespace OpenLoco::Ui::Windows::Construction::Construction
     // 0x0049B50C
     void reset()
     {
-        std::fill(std::begin(_scenarioSignals), std::end(_scenarioSignals), 0xFF);
-        std::fill(std::begin(_scenarioBridges), std::end(_scenarioBridges), 0xFF);
-        std::fill(std::begin(_scenarioTrainStations), std::end(_scenarioTrainStations), 0xFF);
-        std::fill(std::begin(_scenarioTrackMods), std::end(_scenarioTrackMods), 0xFF);
+        std::fill(std::begin(Scenario::getConstruction().signals), std::end(Scenario::getConstruction().signals), 0xFF);
+        std::fill(std::begin(Scenario::getConstruction().bridges), std::end(Scenario::getConstruction().bridges), 0xFF);
+        std::fill(std::begin(Scenario::getConstruction().trainStations), std::end(Scenario::getConstruction().trainStations), 0xFF);
+        std::fill(std::begin(Scenario::getConstruction().trackMods), std::end(Scenario::getConstruction().trackMods), 0xFF);
```

Comment:
Maybe we should introduce a reset function for these in the `Scenario::Construction` struct you've introduced, resetting the entire struct to 0xFF.

Reply:
Agreed. Shall I add reset lines for roadStations and roadMods and are which is missing in ConstructionTab reset function?